Resources/views/Menu/sonata_menu.html.twig

```diff
@@ -33,6 +33,10 @@
             {%- endfor %}
         {%- endif %}
 
+        {%- if item.extra('route') is not empty and item.extra('route') == request.get('_route') %}
+            {%- set active = true %}
+        {%- endif %}
+
```

Comment:
ping @Soullivaneuh so please merge as it is for now

thx @mvhirsch for the review!